src/libraries/System.IO.Pipelines/ref/System.IO.Pipelines.cs

```diff
@@ -95,10 +95,12 @@ public static partial class StreamPipeExtensions
     public partial class StreamPipeReaderOptions
     {
         public StreamPipeReaderOptions(System.Buffers.MemoryPool<byte>? pool = null, int bufferSize = -1, int minimumReadSize = -1, bool leaveOpen = false) { }
+        public StreamPipeReaderOptions(System.Buffers.MemoryPool<byte>? pool, int bufferSize, int minimumReadSize, bool leaveOpen, bool waitForData) { }
```

Comment:
```suggestion
        public StreamPipeReaderOptions(System.Buffers.MemoryPool<byte>? pool, int bufferSize, int minimumReadSize, bool leaveOpen) { }
        public StreamPipeReaderOptions(System.Buffers.MemoryPool<byte>? pool = null, int bufferSize = -1, int minimumReadSize = -1, bool leaveOpen = false, bool waitForData = false) { }
```
This is what I think it should be.

Reply:
Got it!